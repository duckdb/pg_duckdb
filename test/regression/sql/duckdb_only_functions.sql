-- We should be able to use duckdb-only functions even when
-- duckdb.force_execution is turned off
SET duckdb.force_execution = false;

\getenv pwd PWD

select * from read_parquet(:'pwd' || '/data/unsigned_types.parquet') as (usmallint int);
select * from read_parquet(ARRAY[:'pwd' || '/data/unsigned_types.parquet']) as (usmallint int);

select * from read_csv(:'pwd' || '/data/web_page.csv') as (column00 int) LIMIT 2;
select * from read_csv(ARRAY[:'pwd' || '/data/web_page.csv']) as (column00 int) LIMIT 2;

-- TODO: Add a test for scan_iceberg once we have a test table
