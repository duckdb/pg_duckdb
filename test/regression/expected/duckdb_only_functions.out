-- We should be able to use duckdb-only functions even when
-- duckdb.force_execution is turned off
SET duckdb.force_execution = false;
\getenv pwd PWD
select * from read_parquet(:'pwd' || '/data/unsigned_types.parquet') as (usmallint int);
 usmallint 
-----------
     65535
(1 row)

select * from read_parquet(ARRAY[:'pwd' || '/data/unsigned_types.parquet']) as (usmallint int);
 usmallint 
-----------
     65535
(1 row)

select * from read_csv(:'pwd' || '/data/web_page.csv') as (column00 int) LIMIT 2;
 column00 
----------
        1
        2
(2 rows)

select * from read_csv(ARRAY[:'pwd' || '/data/web_page.csv']) as (column00 int) LIMIT 2;
 column00 
----------
        1
        2
(2 rows)

-- TODO: Add a test for scan_iceberg once we have a test table
